docs/api.md

```diff
@@ -318,12 +318,17 @@ Shortcut for [page.mainFrame().click(selector[, options])](#frameclickselector-o
 #### page.emulate(options)
 - `options` <[Object]>
   - `viewport` <[Object]>
-    - `width` <[number]> page width in pixels.
-    - `height` <[number]> page height in pixels.
+    - `width` <[number]> page width in pixels. **required**
+    - `height` <[number]> page height in pixels. **required**
     - `deviceScaleFactor` <[number]> Specify device scale factor (could be thought of as dpr). Defaults to `1`.
     - `isMobile` <[boolean]> Whether the `meta viewport` tag is taken into account. Defaults to `false`.
     - `hasTouch`<[boolean]> Specifies if viewport supports touch events. Defaults to `false`
     - `isLandscape` <[boolean]> Specifies if viewport is in landscape mode. Defaults to `false`.
+    - `defaultBackgroundColor` <[Object]>
```

Comment:
- How about optional transparentBackground instead? I don't think we should expose ugly numeric rbg values.
- I'm not sure how much this is needed, but how about capturing iPhone screenshot with no background? You seem to be making it hard.

I had my own version that had `page.screenshot` accept options.noBackground. Let me upload it.